src/pylhe/__init__.py

```diff
@@ -171,8 +174,8 @@ def readLHE(thefile):
                 for p in particles:
                     particle_objs += [LHEParticle.fromstring(p)]
                 yield LHEEvent(eventinfo, particle_objs)
-    except ET.ParseError:
-        print("WARNING. Parse Error.")
+    except ET.ParseError as e:
```

Comment:
We're currently failing CI here given `flake8`'s [E722 do not use bare 'except'](https://www.flake8rules.com/rules/E722.html) but this isn't the fault of this PR. I'll fix this here as soon as I'm out of my next meeting.

Reply:
Actually this isn't here, but lower down.